src/docs/08-operation-guide/01-setup.md

```diff
@@ -0,0 +1,348 @@
+---
+layout: default
+title: Worker service
+permalink: /docs/operation-guide/setup
+---
+
+# Cluster Setup
```

Comment:
It helps the reader if you are consistent with your title and your main heading. So decide to either use `Worker service` or `Cluster setup` and update the title or heading accordingly.